taskset.go

```diff
@@ -140,23 +141,32 @@ func (ts *TaskSet) SortByResolved(dir SortByDirection) {
 	}
 }
 
+// MustLoadTask is the same as LoadTask, except it exits on error.
+func (ts *TaskSet) MustLoadTask(task *Task) Task {
+	newTask, err := ts.LoadTask(task)
+	if err != nil {
+		ExitFail("%s, task %s", err, task.UUID)
+	}
+	return *newTask
+}
+
 // LoadTask adds a task to the TaskSet, but only if it has a new uuid or no uuid.
 // Return annotated task.
```

Comment:
note that we actually modify the task being provided to us, and return the exact same pointer we got in.